examples/karaf-redis-example/karaf-redis-example-provider/src/main/java/org/apache/karaf/examples/redis/provider/UserServiceImp.java

```diff
@@ -0,0 +1,31 @@
+package org.apache.karaf.examples.redis.provider;
+
+import org.apache.karaf.examples.redis.api.User;
+import org.apache.karaf.examples.redis.api.UserService;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+public class UserServiceImp implements UserService {
```

Comment:
Should be named `UserServiceImpl` to be consistent.

Reply:
Solved